coriolis/osmorphing/windows.py

```diff
@@ -248,3 +287,159 @@ def run_user_script(self, user_script):
         except Exception as err:
             raise exception.CoriolisException(
                 "Failed to run user script.") from err
+
+    def _disable_cloudbase_init(self):
+        key_name = str(uuid.uuid4())
+        self._load_registry_hive(
+            "HKLM\\%s" % key_name,
+            "%sWindows\\System32\\config\\SYSTEM" % self._os_root_dir)
+        try:
+            if self._check_cloudbase_init_exists(key_name):
+                self._event_manager.progress_update(
+                    "Disabling cloudbase-init")
+                self._set_service_start_mode(
+                    key_name, CLOUDBASEINIT_SERVICE_NAME,
+                    SERVICE_START_DISABLED)
+        finally:
+            self._unload_registry_hive("HKLM\\%s" % key_name)
+
+    def _check_cloudbase_init_exists(self, key_name):
+        reg_service_path = (SERVICE_PATH_FORMAT %
+                            (key_name, CLOUDBASEINIT_SERVICE_NAME))
+        return self._conn.exec_ps_command(
```

Comment:
Cloudbase-Init installer installs the service as 'cloudbase-init' (hardcoded).
https://github.com/cloudbase/cloudbase-init-installer/blob/master/CloudbaseInitSetup/Product.wxs#L217

Reply:
Not really. I mean, it would have to be a custom build of ```cloudbase-init```. Also, no real conflicts can take place. Both instances would share the same metadata service. All plugins are idempotent, except for the LocalScripts plugin which can run user defined scripts. But this is an edge case that even edge cases envy.